src/Prover-Axelar.sol

```diff
@@ -84,19 +79,29 @@ contract DealClient is AxelarExecutable {
         pieceStatus[proposal.piece_cid.data] = Status.DealPublished;
         
         int64 duration = CommonTypes.ChainEpoch.unwrap(proposal.end_epoch) - CommonTypes.ChainEpoch.unwrap(proposal.start_epoch);
+        // Expects deal label to be chainId encoded in bytes
+        uint256 chainId = abi.decode(proposal.label.data, (uint256));
         DataAttestation memory attest = DataAttestation(proposal.piece_cid.data, duration, mdnp.dealId, uint256(Status.DealPublished));
         bytes memory payload = abi.encode(attest);
-        call_axelar(payload, proposal.provider.data, AXELAR_GAS_FEE);
+        if (chainId == block.chainid) {
+            IBridgeContract(chainIdToDestinationChain[chainId].destinationAddress)._execute(chainIdToDestinationChain[chainId].chainName, addressToHexString(address(this)), payload);
```

Comment:
nit: idk if just github ui but it looks like the tabbing is off

Reply:
ig its github ui only